README.md

```diff
@@ -30,12 +39,16 @@ Under the current tax laws of of the DRH, taxes are calculated as follows:
 ### Operating System
 
 ### Programming Language
+- Python
 
 ### Additional System Requirements
 
 ### Dependencies
 
 ## Input data requirements
+### Corporate tax calculation
+- Income
+- Incurred loss
```

Comment:
What type of information is this?  A string, list, integers, or decimal numbers like a float.